packages/config/config/devices/0x0086/zw141.json

```diff
@@ -0,0 +1,229 @@
+			"description": "The available range for time set is 1 to 127 seconds",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 127,
+			"defaultValue": 127,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"80": {
+			"label": "Notification To Send to The Associated Devices (Group 1) on State Change",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Send nothing",
+					"value": 0
+				},
+				{
+					"label": "Send basic cc report",
+					"value": 1
+				}
+			]
+		},
+		"85": {
+			"label": "Operation Mode of External Switch",
+			"description": "Refer to the user manual for the description of operation mode 1 or 2",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Operation mode 1",
+					"value": 0
+				},
+				{
+					"label": "Operation mode 2",
+					"value": 1
+				}
+			]
+		},
+		"120": {
+			"label": "External Switch Mode For S1",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 4,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Unidentified mode",
+					"value": 0
+				},
+				{
+					"label": "Reserved",
+					"value": 1
+				},
+				{
+					"label": "3-way switch mode",
+					"value": 2
+				},
+				{
+					"label": "Momentary push button mode",
+					"value": 3
+				},
+				{
+					"label": "Enter automatic identification mode",
+					"value": 4
+				}
+			]
+		},
+		"121": {
+			"label": "External Switch Mode For S2",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 4,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Unidentified mode",
+					"value": 0
+				},
+				{
+					"label": "Reserved",
+					"value": 1
+				},
+				{
+					"label": "3-way switch mode",
+					"value": 2
+				},
+				{
+					"label": "Momentary push button mode",
+					"value": 3
+				},
+				{
+					"label": "Enter automatic identification mode",
+					"value": 4
+				}
+			]
+		},
+		"248": {
+			"label": "S1/S2 Functionality",
+			"valueSize": 1,
+			"minValue": -128,
+			"maxValue": 4,
+			"defaultValue": 83,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "The setting for sending nif, entering rf power level test mode and factory reset are effective",
+					"value": -128
+				},
+				{
+					"label": "The functions of s1 and s2 are disabled",
```

Comment:
In any event, please shorten these labels. "S1 and S2 disabled." "Sending NIF enabled" etc.